docs/source/project-management/sdc-sit-plan.rst

```diff
@@ -0,0 +1,63 @@
+.. _sdc-sit-plan:
+
+SDC Software Integration Test (SIT) Plan
+========================================
+
+The SDC test plan is based on the `SOC Test Plan <https://lasp.colorado.edu/galaxy/display/IMAP/IMAP+SOC+Test+Plan>`_, but makes the below customizations to help fit the needs of SDC specific testing. Planned SIT dates and planned functionality tested can also be found in the linked SOC Test Plan.
+
+Software Inegration Test Timeline
+---------------------------------
+====================== =============================================================================================================================
+Timeline               Step
+====================== =============================================================================================================================
+Start of development   SDC creates a SIT procedures page and adds/updates test steps throughout the development process
+~4 weeks prior to test SDC begins to finalize test steps
+~2 weeks prior to test SDC conducts a dry run of the test procedures, makes any necessary fixes, test step updates
+~1 week prior to test  SDC conducts a Test Readiness Review (TRR) to review the final version of the test procedures 
+Test                   SDC conducts the SIT
+~1 week after test     SDC reviews all failures and proposed fixes, decides on time for re-run of failed steps, and updates requirement test status
+~2 weeks after test    SDC finalizes the test report
+====================== =============================================================================================================================
+
+Start of development
+^^^^^^^^^^^^^^^^^^^^
+
+Over the course of development, the SDC adds steps to the SIT test procedures. This requires a coordinated effort among all SDC developers to document test steps as new features are introduced. Taking an incremental approach allows for better documentation as more complexity is added throughout the course of development. 
```

Comment:
I'd recommend linking to your requirements document here... although might be tricky since it isn't merged.